docs/docsite/rst/reference_appendices/YAMLSyntax.rst

```diff
@@ -107,7 +107,10 @@ While in the above ``>`` example all newlines are folded into spaces, there are
         d
           e
         f
-    same_as: "a b\nc d\n  e\nf\n"
+    
+same_as::
```

Comment:
@sankur-codes 
Thanks for your contribution. 

- As bcoca says, we need to preserve the line that defines the key:value pair  `same_as: a b\nc d\n  e\nf\n`.  
- However, it is a good idea to split the 2 examples (`fold_some_newlines` and `same_as`) into separate code blocks.